### DISCUSSION TITLE
Uncaught (in promise) Error: Module "./@apollo/client/link/context" does not exist in container

### DISCUSSION THREAD

**Discussion by caoxuccc** [umijs/umi#6884] [Q&A]
项目中使用apollo-client，每次项目启动，或者切换分支后，都会报错，目前解决办法是删掉.umi文件后重新编译，可恢复，这个是啥问题😂

![image](https://user-images.githubusercontent.com/51979013/124529461-4102b580-de3d-11eb-9c74-30171230c500.png)

**Comment by magi-wang** [comment]
遇到同样的问题

**Comment by petrov-ro** [comment]
Remove .umi folder from src directory.
